### PR TITLE
Use correct identity for ELN

### PR DESCRIPTION
**Pull request by sgallagher**
Resolves: rhbz#1909045

Signed-off-by: Stephen Gallagher <sgallagh@redhat.com>

### CODE REVIEW

File: data/product.d/fedora-eln.conf [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Anaconda configuration file for Fedora ELN.
 
 [Product]
-product_name = Fedora
-variant_name = ELN
+product_name = Fedora-ELN
+variant_name = Everything
```
Comment (poncovka) [inline data/product.d/fedora-eln.conf:5]: The variant name doesn't have to be specified when it is not necessary.

**Comment by poncovka** [comment]
`fedora_products_test` needs to be updated to fix the tests.

File: data/product.d/fedora-eln.conf (hunk 1 above)
Comment (sgallagher) [reply data/product.d/fedora-eln.conf:5]: How does one determine whether it's necessary?

Comment (jkonecny12) [reply data/product.d/fedora-eln.conf:5]: It's not necessary because you want to have this configuration for all the variants.

**Review by jkonecny12 — APPROVED** [review]
LGTM! Thanks!

**Comment by jkonecny12** [comment]
After the tests will be fixed of course :)

**Review by poncovka — APPROVED** [review]
Looks good to me.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by rvykydal — APPROVED** [review]
Looks good to me.

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Comment by jkonecny12** [comment]
Tested, works great! Thanks a lot @sgallagher for all your work!

**Pull request merged**